components/Form/Checkboxes/Checkboxes.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { ChangeEvent, useMemo } from "react";
+import clsx from "clsx";
+// import { useController, useFormContext } from "react-hook-form";
+
+import styles from "./Checkboxes.module.scss";
+import StyledCheckbox from "./StyledCheckbox/StyledCheckbox";
+import RadioButton from "./RadioButton/RadioButton";
+import ErrorMessage from "../ErrorMessage/ErrorMessage";
+import { useController, useFormContext } from "react-hook-form";
+import TextInput from "../TextInput/TextInput";
+import StyledInput from "../StyledInput/StyledInput";
+
+export type CheckboxOption = {
+    label: string | number;
+    value: string | number;
+
+    // `isRadio: true` indicates that this specific checkbox will act like a radio button
+    // (i.e. if this is checked, then no other checkbox may be also checked)
+    isRadio?: boolean;
+
+    // `isRadioButton: true` indicates that this specific checkbox is a radio button
+    // and the css will change accordingly
+    isRadioButton?: boolean;
```

Comment:
What is the difference between isRadio and isRadioButton?

Reply:
There's no difference -- this was part of the original Checkboxes component from site-2024. I'll merge this and isRadio